smartcontactmanager/src/main/java/com/scm/smartcontactmanager/controller/contact/response/ContactResponseDto.java

```diff
@@ -0,0 +1,5 @@
+package com.scm.smartcontactmanager.controller.contact.response;
+
+public class ContactResponseDto {
```

Comment:
Add a comment line to this class saying you would add changes later.

Reply:
fixed please review